config/src/config/network_config.rs

```diff
@@ -389,6 +391,22 @@ impl Default for NetworkMonitoringConfig {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct NodeMonitoringConfig {
+    pub node_info_request_interval_ms: u64, // The interval (ms) between node info requests
+    pub node_info_request_timeout_ms: u64,  // The timeout (ms) for each node info request
+}
+
+impl Default for NodeMonitoringConfig {
+    fn default() -> Self {
+        Self {
+            node_info_request_interval_ms: 20_000, // 20 seconds
```

Comment:
is it required for node_info_request_interval_ms > node_info_request_timeout_ms for check to work correctly? If so , add a comment for that.